rocketpy/rocket/rocket.py

```diff
@@ -22,6 +22,7 @@
 from rocketpy.tools import parallel_axis_theorem_from_com
 
 
+# pylint: disable=too-many-instance-attributes, too-many-public-methods, too-many-instance-attributes
```

Comment:
I agree! Thank you!
Imagine the case a class have 21 arguments. It's a public class, what can we do about it? a: Nothing! 

Initially I had avoided these errors by increasing the threshold to something unfeasibly, like 50!

But @phmbressan asked to changed that saying that "it is better to know which classes are beating the pylint limits rather than ignoring the rule in the whole code."

I'm sorry but I don't feel like changing this again. I think we should live with these ugly pylint disable message for a while, and then start removing them one by one.

Reply:
Sure, we can deal with this later no problem

But just to emphasize my point, I don't see these ever being useful for our code. Knowing which classes pass the limit does not mean much if the limit is such an arbitrary value